core/trino-main/src/test/java/io/trino/sql/ir/optimizer/TestSimplifyContinuousInValues.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.sql.ir.optimizer;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.trino.sql.ir.Between;
+import io.trino.sql.ir.Constant;
+import io.trino.sql.ir.Expression;
+import io.trino.sql.ir.In;
+import io.trino.sql.ir.IsNull;
+import io.trino.sql.ir.Reference;
+import io.trino.sql.ir.optimizer.rule.SimplifyContinuousInValues;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static io.trino.spi.type.BigintType.BIGINT;
+import static io.trino.spi.type.DoubleType.DOUBLE;
+import static io.trino.spi.type.RealType.REAL;
+import static io.trino.sql.ir.IrUtils.or;
+import static io.trino.testing.TestingSession.testSession;
+import static java.lang.Float.floatToRawIntBits;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestSimplifyContinuousInValues
+{
+    @Test
+    void test()
```

Comment:
Add a test with repeated values. E.g.,

```
x IN (1, 2, 2, 3)
```

Reply:
Added a test, although the rule doesn't simplify this case currently. In actual queries, we get a de-duplicated list when we reach here.

Reply:
I'm aware. Just wanted to have it for completeness, and in case the behavior changes in the future.